src/client/sound_effect.hpp

```diff
@@ -1,21 +1,29 @@
+/*
+** EPITECH PROJECT, 2024
+** R-Type
+** File description:
+** sound_effect
+*/
+
 #pragma once
 
 #include "registry.hpp"
 #include <SFML/Audio.hpp>
 #include <vector>
 #include <string>
+#include <memory>
 
-namespace rtype::game {
+namespace ecs {
 
     /**
      * @brief Class that contains the sound of the game
      * 
      * @tparam Sound_game
      */
-    class Sound_game {
+    class SoundEffect {
         public:
-            Sound_game();
-            ~Sound_game();
+            SoundEffect() = default;
+            ~SoundEffect() = default;
```

Comment:
The explicit defaulting of the constructor and destructor is somewhat redundant.